5g-control-plane/templates/configmap-udr.yaml

```diff
@@ -12,7 +12,7 @@
 {{- $sbi := index $config "sbi" }}
 
 {{- if not (hasKey $config "mongodb") -}}
-{{- $_ := dict "name" .Values.config.mongodb.name "url" .Values.config.mongodb.url | set $config "mongodb" -}}
+{{- $_ := dict "name" .Values.config.mongodb.name "url" .Values.config.mongodb.url "authKeysDbName" .Values.config.mongodb.authKeysDbName "authUrl" .Values.config.mongodb.authUrl | set $config "mongodb" -}}
```

Comment:
Have provided comments to other code.

Reply:
If you are referring to Canonical, no because they do not use Helm Charts. They have a different way to deploy the SD-Core